ethcore/src/account.rs

```diff
@@ -46,8 +45,6 @@ pub struct Account {
 	code_size: Option<u64>,
 	// Code cache of the account.
 	code_cache: Arc<Bytes>,
-	// Account is new or has been modified.
-	filth: Filth,
```

Comment:
Dirty status is tracked in the State now. This is done to simplify code that tracks dirty status for missing/killed accounts and to track all accounts that were modified in the block.